backend/protzilla/methods/data_analysis.py

```diff
@@ -185,6 +202,49 @@ class DifferentialExpressionANOVA(DataAnalysisStep):
         "filtered_proteins",
     ]
 
+    def create_form(self):
+        return Form(
+            label="ANOVA",
+            input_fields=[
+                DropdownField(
+                    name="protein_df", label="Step to use protein intensities from"
+                ),
+                DropdownField(
+                    name="multiple_testing_correction_method",
+                    label="Multiple testing correction",
+                    options=MultipleTestingCorrectionMethod,
+                    value=MultipleTestingCorrectionMethod.benjamini_hochberg,
+                ),
+                FloatField(
+                    name="alpha",
+                    label="Error rate (alpha)",
+                    value=0.05,
+                    min=0,
+                    max=1,
+                    step=0.01,
+                    separatePrefix="\u03B1",
+                ),
+                DropdownField(name="grouping", label="Grouping from metadata"),
+                DropdownField(
```

Comment:
Use `MultiSelectField` instead of `DropdownField` here, because the backend expects a list rather than a single string.
```suggestion
                MultiSelectField(
```